Libraries/LibGfx/ImageFormats/PNGLoader.cpp

```diff
@@ -334,6 +339,15 @@ ErrorOr<size_t> PNGLoadingContext::read_frames(png_structp png_ptr, png_infop in
                 VERIFY_NOT_REACHED();
             }
         }
+
+        frame_count = animation_frame_count;
+
+        // If we didn't find any valid animation frames with fcTL chunks, fall back to using the base IDAT data as a single frame.
```

Comment:
This comment confused me, I thought that it would be the only path where an `IDAT` chunk's data could be included in `frame_descriptors`.

Reply:
Oh, right, I shoud've been more clear. It's for the case like this: IHDR → acTL → IDAT → IEND. So techlically it's APNG, but without any animation frames. Other browsers just display base IDAT image in this case, so I added this check

Reply:
Added the spec quote 🫡